packages/asymptote/build.sh

```diff
@@ -2,12 +2,18 @@ TERMUX_PKG_HOMEPAGE=https://asymptote.sourceforge.io/
 TERMUX_PKG_DESCRIPTION="A powerful descriptive vector graphics language for technical drawing"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=(2.89)
-TERMUX_PKG_VERSION+=(1.0.1)
-TERMUX_PKG_SRCURL=(https://downloads.sourceforge.net/asymptote/asymptote-${TERMUX_PKG_VERSION}.src.tgz
-                   https://github.com/g-truc/glm/archive/${TERMUX_PKG_VERSION[1]}.tar.gz)
-TERMUX_PKG_SHA256=(f64e62b4ee4f85f1a78640c4f1e8a6f98e91f54edacab19727c7cabe94a57f5b
-                   9f3174561fd26904b23f0db5e560971cbf9b3cbda0b280f04d5c379d03bf234c)
+TERMUX_PKG_VERSION=(
+	2.89
+	1.0.1
+)
+TERMUX_PKG_SRCURL=(
+	https://downloads.sourceforge.net/asymptote/asymptote-${TERMUX_PKG_VERSION}.src.tgz
```

Comment:
Explicitly use index `[0]` to show that it's an array.
<sup>Also I'm pretty sure Shellcheck yells at you for not indexing on arrays</sup>
```suggestion
	https://downloads.sourceforge.net/asymptote/asymptote-${TERMUX_PKG_VERSION[0]}.src.tgz
```